.github/workflows/publish-canary.yml

```diff
@@ -47,6 +47,6 @@ jobs:
           pnpm changeset pre enter ${date}
           pnpm changeset version
           pnpm changeset pre exit
-          pnpm changeset publish --tag ${{ inputs.tag }}
+          pnpm changeset publish --tag ${{ github.event_name == 'schedule' && 'canary' || inputs.tag }}
```

Comment:
How about
```
pnpm changeset publish --tag ${{ github.event_name == 'schedule' ? 'canary' : inputs.tag }}
```

Reply:
Because I'm not sure if the conditional logic you applied works properly in github actions

Reply:
I guess my suggestion comes a little late 😬

Reply:
To my understanding the way I did it will work in GH workflows but the operators you suggested don't exist in GH workflows.

Reply:
=> https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/expressions#example

Reply:
Hmm good point, I guess you followed the[ example for their ternary equivalent](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/expressions#example), makes sense.